doc/qbk/1.1.quicklook.qbk

```diff
@@ -13,6 +13,8 @@ This section is intended to give the reader a brief overview of the features
 and interface style of the library.
 
 Say you have the following URL that you want to parse:
+
+[teletype]
```

Comment:
why `[teletype]`? This should not be needed

Reply:
Without teletype, quickbook assumes this is c++. This generates a warning and highlights the syntax incorrectly.

Reply:
Ok so it is just for the non-c++ blocks. makes sense

Reply:
Yes. This command tells quickbook the next block is plain text.

Reply:
Quickbook chose the worst keyword possible for that here, but I have no power over that.

Reply:
we need this change